### ISSUE TITLE
shebang line in rebuild-iptables script is not system-independent

### ISSUE THREAD

**Issue by billywatson** [chef-cookbooks/iptables#18]
There's a semi-complicated bit of code to apply a different shebang line depending on where ruby is located in the system. IMO, it should just use

```
#!/usr/bin/env ruby
```

**Comment by pghalliday** [comment]
Indeed, currently this doesn't work when using chefdk :(

**Comment by pghalliday** [comment]
I found another case where it broke - I installed ruby from source on an ARM device and it was installed to /usr/local/bin/ruby. I don't want to change it now as it is in my base image which takes hours to build. Also as there is no Chef omnibus installer for ARM devices I installed chef as a Gem. Thus I have no embedded ruby.

The result is now that I want to configure some port forwarding on a number of devices with this cookbook, it doesn't work because ruby is in neither of the 2 places it looks. It is registered to work correctly with `/usr/bin/env ruby` though.

An attribute or the last solution from @billywatson suggested in the closed pull request #19 would work I think
